components/toolbar/test/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Toolbar from '../';
+
+describe( 'Toolbar', () => {
+	describe( 'basic rendering', () => {
+		it( 'should render an empty node controls are not passed', () => {
+			const toolbar = shallow( <Toolbar /> );
+			expect( toolbar.type() ).to.be.null();
+		} );
+
+		it( 'should render an empty node controls are empty', () => {
+			const toolbar = shallow( <Toolbar controls={ [] } /> );
+			expect( toolbar.type() ).to.be.null();
+		} );
+
+		it( 'should render a list of controls with IconButtons', () => {
+			const clickHandler = ( event ) => event;
+			const controls = [
+				{
+					icon: 'wordpress',
+					title: 'WordPress',
+					subscript: 'wp',
+					onClick: clickHandler,
+					isActive: false,
+				},
+			];
+			const toolbar = shallow( <Toolbar controls={ controls } /> );
+			const listItem = toolbar.find( 'IconButton' );
+			expect( toolbar.type() ).to.equal( 'ul' );
+			expect( listItem.props().icon ).to.equal( 'wordpress' );
```

Comment:
Yeah, I thought I updated these, guess not lol. Will change.  Should it be changed to the one command style?

```
expect( listItem.props() ).to.eql( {
	// ...keys
} );
```